utils/daily.go

```diff
@@ -0,0 +1,89 @@
+package utils
+
+import (
+	"cronicle/ui/constants"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+type WriteDailyParams struct {
+	Message, Tags, Date string
+}
+
+func GetDate() string {
+	time := time.Now()
+	return time.Format("2006-01-02")
+}
+
+func AddToCurrentDaily(w WriteDailyParams) {
```

Comment:
It'd be nice if we could have default params. Then this function could be `AddToDailyLog` and it would take a date as a param. If date is not given, we'd default to today.